src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -271,7 +271,7 @@ private void SetupCommonOptionsForCommands(List<Command> commands)
             }
 
             int numOfFilesAnalyzed = exitCodes.Where(x => x == ExitCode.Success || x == ExitCode.Violation).Count();
-            Console.WriteLine($"{Environment.NewLine}Analyzed {numOfFilesAnalyzed} {(numOfFilesAnalyzed == 1 ? "file" : "files")}.");
+            Console.WriteLine($"{Environment.NewLine}Analyzed {numOfFilesAnalyzed} {(numOfFilesAnalyzed == 1 ? "file" : "files")} in directory.");
```

Comment:
```suggestion
            Console.WriteLine($"{Environment.NewLine}Analyzed {numOfFilesAnalyzed} {(numOfFilesAnalyzed == 1 ? "file" : "files")} in the directory specified.");
```

Reply:
Or
```suggestion
            Console.WriteLine($"{Environment.NewLine}Analyzed {numOfFilesAnalyzed} {(numOfFilesAnalyzed == 1 ? "file" : "files")} in directory {directoryPath}.");
```
?

Reply:
The directory path is already printed on top

Reply:
I had target directory first and removed it but I think Vera's suggestion is better than target.